### ISSUE TITLE
Desktop apps like Cordova ?

### ISSUE THREAD

**Issue by joeblew99** [manifoldjs/ManifoldJS#221]
Hey everyone,

This looks interesting for me as I already use manifests for my front-end html projects.

Does this allow to build desktop apps, like how Cordova does ? For example osx, Ubuntu and windows 10.

I got the impression it does not.

Please clarify ..

**Comment by boyofgreen** [comment]
Hey @joeblew99   We do have a few early versions of desktop:
•	Windows 7+: https://github.com/sozercan/manifoldjs-win32
•	macOS 10.9+: https://github.com/sozercan/manifoldjs-mac 
•	Ubuntu 12.04+ (and others): https://github.com/sozercan/manifoldjs-linux

we are actaully working to refine some of these right now, and make them official platforms.  Starting with MacOS.  Which ones do you need?  Maybe we can prioritize.

**Comment by joeblew99** [comment]
hey @boyofgreen 

thanks for the update. Sounds really promising. 

my team is supporting all 3 desktops. But in order of priority its:
1. osx
2. windows
3. ubuntu (other flavours of linux later or not at all)

Its great that your supporting windows 7 and up too. Many users still stuck on it.

**Comment by joeblew99** [comment]
i just had a look at the OSX one.
Seems your using electron prebuilt ?
I also noticed that electron prebuilt is being used for all the other Desktops (windows and Ubuntu)

This is good news for me ( at least), as i depend on Shadow DOM and other things from Chromium).

**Comment by joeblew99** [comment]
i tried the OSX version on my Polymer app that uses web manifests. 
Worked first time, which is great

From OSX; the linux one worked too.
From OSX, the windows one did not work.
Raised an issue here: https://github.com/sozercan/manifoldjs-win32/issues/1

**Comment by joeblew99** [comment]
closing, since your working on it :)

**Comment by boyofgreen** [comment]
let me look into it, but I think you can only build Mac on the Mac and PC on the PC, and Linux anywhere.  I'll get details soon from the dev who is working on it.  We'll have a project stood up in manifoldjs soon where we can put issues

**Comment by joeblew99** [comment]
ok thanks. i was wondering the same.

**Comment by boyofgreen** [comment]
I can confirm the Mac will only build on the Mac.  testing PC now

**Comment by joeblew99** [comment]
@boyofgreen 
Ok thanks.
I got Linux to build on Mac.
I did not get windows to build on mac.


This is off topic, but as i understand it it does not save the web site at build time in a persistent way.
Its more like at run time it calls out to a web site to get the content.
Sorry if i sounds confused, but well i am a little.

For my use case, i really want to be able to point it at a web site at build time and have it sucked in. Then its fixed, unless i do another deploy.

Maybe there is a half way house here too.
Again sorry for the open ended question.
